.changeset/purple-dancers-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
+---
+
+You can now filter orders by IDs of orders. Input field can handle one or multiple IDs, just use a comma, space or Enter to separate them.
```

Comment:
_Filters for orders now allow filtering by ID, This means its form handles one or multiple ID that you can attach by using comma, space, or enter for separation. _